content/22.vaccines.md

```diff
@@ -2,8 +2,20 @@
 
 ### Abstract
 
+The release of the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) genome sequence on January 10th, 2020, mobilised global vaccine development on a scale never before seen in response to the coronavirus disease 2019 (COVID-19) pandemic. 
```

Comment:
Please correct me if I'm wrong, but my understanding is that not all the vaccines used the genome sequence, but this sequence allowed for the development of novel vaccines that target SARS-CoV2. If that's the case I might re-word this to focus on the effort towards rapid vaccine development in response to the pandemic without mentioning the sequencing...

Reply:
```suggestion
The release of the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) genome sequence on January 10th, 2020, mobilized global vaccine development on a scale never before seen in response to the coronavirus disease 2019 (COVID-19) pandemic. 
```

Reply:
Again apologies for the Irish spelling haha